### PR TITLE
Fixing env variables and README

### PR DESCRIPTION
**Pull request by teresaqhoang**
### Motivation and Context

Fixing environment variables to prefix required "REACT_APP" so the CRA tool knows to use it during the build process

### Contribution Checklist

- [ ] The code builds clean without any errors or warnings
- [ ] The PR follows SK Contribution Guidelines (https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md)
- [ ] The code follows the .NET coding conventions (https://learn.microsoft.com/dotnet/csharp/fundamentals/coding-style/coding-conventions) verified with `dotnet format`
- [ ] All unit tests pass, and I have added new tests where possible
- [ ] I didn't break anyone :smile:

### CODE REVIEW

File: samples/apps/copilot-chat-app/WebApp/README.md [hunk 1]
```diff
@@ -4,31 +4,34 @@
...
+The Copilot Chat sameple showcases how to build an enriched intelligent app, with multiple dynamic components, including command messages, user intent, and context.  The chat prompt will evolve as the conversation between user and application proceeds. 
      
 ### Watch the Copilot Chat Sample Quick Start Video [here](https://aka.ms/SK-Copilotchat-video).
 
-## Running the sample
+## Requirements to run this app
+1. Azure Open AI Service Key and working End point. (https://learn.microsoft.com/en-us/azure/cognitive-services/openai/quickstart?tabs=command-line&pivots=programming-language-studio)
+2.	A registered App in Azure Portal (https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app)
+   -	Select Single-page application (SPA) as platform type, and the Redirect URI will be https://localhost:3000
+   - Select **`Accounts in this organizational directory only`** as the supported account type for this sample.
```
Comment (adrianwyatt) [inline samples/apps/copilot-chat-app/WebApp/README.md:18]: Why not also personal accounts?

File: samples/apps/copilot-chat-app/WebApp/README.md [hunk 2]
```diff
@@ -4,31 +4,34 @@
...
 
-5. **Run** the following command `yarn install` (if you have never run the app before)
-   and/or `yarn start` from the command line.
-6. A browser will automatically open, otherwise you can navigate to `http://localhost:3000/` to use the ChatBot.
+1. Ensure the SKWebApi running at `https://localhost:40443`
+2. Create an **[.env](.env)** file to this folder root with the following variables and fill in with your information, where
+   `REACT_APP_CHAT_CLIENT_ID=` is the GUID copied from the **Application (client) ID** from your app registration in the Azure Portal, 
+   `REACT_APP_BACKEND_URI=` is the URI where your backend is running, and `REACT_APP_TENANT_ID` is your tenant ID. 
+      
+      REACT_APP_BACKEND_URI=https://localhost:40443
```
Comment (glahaye) [inline samples/apps/copilot-chat-app/WebApp/README.md:29]: Looks like these all show up on the same line when rendered as MD

Comment (teresaqhoang) [reply samples/apps/copilot-chat-app/WebApp/README.md:29]: Fixed!

File: samples/apps/copilot-chat-app/WebApp/README.md [hunk 3]
```diff
@@ -1,34 +1,41 @@
...
-
-### Authentication in this sample
-This sample uses the Microsoft Authentication Library (MSAL) for React to sign in users. Learn more about it here: https://learn.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-react.
\ No newline at end of file
+1. Ensure the SKWebApi is running at `https://localhost:40443`. See [SKWebApi README](../SKWebApi/README.md) for instructions.
+2. Create an **[.env](.env)** file to this folder root with the following variables and fill in with your information, where
+   `REACT_APP_CHAT_CLIENT_ID=` is the GUID copied from the **Application (client) ID** from your app registration in the Azure Portal and
+   `REACT_APP_BACKEND_URI=` is the URI where your backend is running.
+
+   REACT_APP_BACKEND_URI=https://localhost:40443
```
Comment (glahaye) [inline samples/apps/copilot-chat-app/WebApp/README.md:32]: Please add a trailing /

The way we concatenate the backend host:port with the path chokes if there is none.

We should probably look into that (I suspect it's currently just a string concatenation) but at least having the app working with the example data would be a start

**Review by glahaye — APPROVED** [review]
Approved with a small suggestion

**Review by adrianwyatt — APPROVED** [review]

**Pull request merged**